jiant/proj/main/modeling/model_setup.py

```diff
@@ -162,7 +156,7 @@ def load_encoder_from_transformers_weights(
     remainder_weights_dict = {}
     load_weights_dict = {}
     model_arch = ModelArchitectures.from_encoder(encoder=encoder)
-    encoder_prefix = MODEL_PREFIX[model_arch] + "."
+    encoder_prefix = model_arch.value + "."
```

Comment:
`f"{model_arch.value}."

Reply:
seems a less readable (the period could be overlooked)

Reply:
Ah, fair enough.